src/core/InputDropdown/style.ts

```diff
@@ -202,7 +202,7 @@ const isDisabled = (props: InputDropdownProps): SerializedStyles => {
   `;
 };
 
-const doNotForwardProps = ["intent", "open", "sdsStage", "sdsStyle"];
```

Comment:
since `InputDropdownProps` does include `sdsStage` and `sdsStyle`, I think it makes sense to keep these two strings here?

```ts
export interface InputDropdownProps extends CommonThemeProps {
  disabled?: boolean;
  intent?: "default" | "error" | "warning";
  label: string;
  onClick: (event: React.MouseEvent<HTMLElement>) => void;
  open?: boolean;
  sdsStage: "default" | "userInput";
  sdsStyle?: "minimal" | "square" | "rounded";
  sdsType?: "singleSelect" | "multiSelect";
  details?: string;
  counter?: string;
}
```

Reply:
Fixed ^_^